golem-cluster-dev.yaml

```diff
@@ -30,7 +30,7 @@ provider:
     node_config:
       # if not provided, image_tag will be autodetected based on currently used python and ray versions
       # check available versions at https://registry.golem.network/explore/golem/ray-on-golem
-      image_tag: "lucjandudek/ray-on-golem:py3.10.12-ray2.3.1-coremain"
+      image_tag: "golem/ray-on-golem:0.1.2-py3.10.13-ray2.3.1"
```

Comment:
Are you sure `golem/ray-on-golem:0.1.2-py3.10.13-ray2.3.1` has the same golem-core dependencies as the golem-core:main?